addon/webextension/background/main.js

```diff
@@ -108,11 +110,6 @@ window.main = (function () {
     title: browser.i18n.getMessage("contextMenuLabel"),
     contexts: ["page"],
     documentUrlPatterns: ["<all_urls>"]
-  }, () => {
```

Comment:
I think this catcher is the only way to catch an error from contextMenu, unlike some of the other uses of `lastError`.